data/appstream/io.itsterminal.WebPConverter.appdata.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright [2024] [Michael Scuteri] -->
+<component type="desktop-application">
+  <id>io.itsterminal.WebPConverter</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-3.0</project_license>
+  <name>WebP Converter</name>
+  <summary>A tool to convert images to WebP format.</summary>
+  <developer id="io.itsterminal">
+    <name>Michael Scuteri</name>
+  </developer>
+  <description>
+    <p>WebP Converter allows you to convert images to the WebP format, while allowing you to control the compression qaulity</p>
+  </description>
+  <launchable type="desktop-id">io.itsterminal.WebPConverter.desktop</launchable>
+  <categories>
+    <category>Graphics</category>
+    <category>Utility</category>
+  </categories>
+  <icon type="stock">io.itsterminal.WebPConverter</icon>
+  <branding>
+    <color type="primary" scheme_preference="light">#ffffff</color>
+    <color type="primary" scheme_preference="dark">#0c247a</color>
+  </branding>
+  <content_rating type="oars-1.1" />
+  <url type="homepage">https://itsterminal.io/</url>
+  <screenshots>
+    <screenshot type="default">
+        <image>https://raw.githubusercontent.com/MichaelScuteri/webp-converter-gnome/refs/heads/main/screenshots/webp-converter.png</image>
```

Comment:
use a commit instead of `refs/heads/main`

Reply:
updated